docs/source/conf.py

```diff
@@ -21,126 +21,126 @@
 #
 # Auto generate API documentation
 #
-api_source_dir = os.path.join(root_path, 'avocado')
-base_api_output_dir = os.path.join(root_path, 'docs', 'source', 'api')
-try:
-    apidoc = path.find_command('sphinx-apidoc')
-    apidoc_template = apidoc + " -o %(output_dir)s %(api_source_dir)s %(exclude_dirs)s"
-except path.CmdNotFoundError:
-    apidoc = False
+#api_source_dir = os.path.join(root_path, 'avocado')
+#base_api_output_dir = os.path.join(root_path, 'docs', 'source', 'api')
+#try:
+#    apidoc = path.find_command('sphinx-apidoc')
+#    apidoc_template = apidoc + " -o %(output_dir)s %(api_source_dir)s %(exclude_dirs)s"
+#except path.CmdNotFoundError:
+#    apidoc = False
```

Comment:
If we are not going to use the code anymore, wouldn't it be better to delete it instead of commenting?

Reply:
Actually I forgot to bring the API back. We need this code. I will work on this. Nice catch. Thanks.